src/components/organisms/AssetContent/MetaMain.tsx

```diff
@@ -65,7 +64,9 @@ export default function MetaMain(): ReactElement {
           Published By{' '}
           <Publisher
             account={owner}
-            verifiedServiceProviderName={verifiedServiceProviderName}
+            verifiedServiceProviderName={
+              isServiceSelfDescriptionVerified ? author : undefined
```

Comment:
This now can lead to an undefined, whilst previously the verifiedServiceProviderName was used. Shouldn't that be used instead of undefined?

Reply:
Fixed!